src/applications/mhv/medical-records/containers/LandingPage.jsx

```diff
@@ -261,8 +261,10 @@ const LandingPage = () => {
                     What if I can’t find all my allergy records?
                   </h3>
                   <p className="vads-u-margin-bottom--2">
-                    To find health information you entered yourself, go to VA
-                    Blue Button on the My HealtheVet website.
+                    This tool only includes health information your VA providers
+                    have entered. To find health information you entered
+                    yourself, go to your VA Blue Button® report on the My
+                    HealtheVet website.
```

Comment:
This content should on two lines. In Sketch:
<img width="604" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/72c3e181-50b5-4850-88d5-236ec276816e">